app/controllers/health_controller.rb

```diff
@@ -0,0 +1,16 @@
+# HealthController contains endpoints that are relevant for checking the status
+# of either portus, or of the components relevant to it.
+class HealthController < ActionController::Base
+  protect_from_forgery with: :exception
+
+  # Simple ping.
+  def index
+    render nothing: true, status: 200
+  end
+
+  # Renders a JSON with the status of each component.
+  def health
+    response, success = ::Portus::Health.check
+    render json: response, status: success ? :ok : :service_unavailable
```

Comment:
In this case we consider service as unavailable only if Clair was enabled but not available. I know that it's a bit strict, but people accessing this endpoint are doing it for either of these two reasons:

1. They want to get a JSON response.
2. They want to make sure that either everything is working, or otherwise there's a problem, so some service is unavailable.

If they just care that things are more or less working, then they would use `/_ping`.